File: Course_01/C++ Language/EstruturaSequencial/pagamento.cpp
#include <iostream>
#include <string>
#include <iomanip>

using namespace std;

int main()
{
    string nome;
    double valorHora, pagamento;
    int horas;

    cout << "Nome: ";
    getline(cin, nome);

    cout << "Valor por hora: ";
    cin >> valorHora;

    cout << "Horas trabalhadas: ";
    cin >> horas;

    pagamento = (double) valorHora * horas;

    cout << fixed << setprecision(2);
    cout << "O pagamento para " << nome << " deve ser " << pagamento << endl;
    return 0;
}

File: Course_01/Python/EstruturaSequencial/pagamento.py
nome = input("Nome: ")
valorHora = float(input("Valor por hora: "))
horasTrabalhadas = int(input("Horas trabalhadas: "))

pagamento = float(valorHora * horasTrabalhadas)

print(f"O pagamento para {nome} deve ser de R$ {pagamento:.2f}")